com.unity.robotics.urdf-importer/Runtime/Controller/Controller.cs

```diff
@@ -130,6 +121,11 @@ void DisplaySelectedJoint(int selectedIndex)
         /// <param name="jointIndex">Index of the link selected in the Articulation Chain</param>
         private void UpdateDirection(int jointIndex)
         {
+            if (jointIndex < 0 || jointIndex >= articulationChain.Length) 
```

Comment:
same as above, this seems not necessary because of the new method SetSelectedJointIndex()?

Reply:
same answer as above :)